src/tokenizer.rs

```diff
@@ -409,6 +415,36 @@ fn parse_whitespace<'a>(current: &'a str, position: &SourcePosition) -> Result<A
     advance(current, position, &PATTERN_WHITESPACE)
 }
 
+fn parse_multi_line_comment<'a>(current: &'a str, position: &SourcePosition) -> Result<(AdvanceResult<'a>, Comment<'a>), AdvanceError> {
+    if let Some(captures) = PATTERN_MULTI_LINE_COMMENT_START.captures(current) {
+        let start_capture = captures.get(0).unwrap();
+        let rest = &current[start_capture.end()..];
+
+        let depth = captures.get(1).unwrap().as_str().len() as u32;
+
+        let reg_str = format!("(.*?)\\]={{{}}}\\]", depth); // Expands to something like (.*?)\]={5}\]
```

Comment:
Can we skip using a capture here and just slice into the string using the start of the match range as the end of the slice?

Also, you can use a raw string literal, like `r"ahh"` to avoid needing to backslach-escape backslashes in the string, which should help with readability.

Reply:
No sweat.